tests/main/auto-refresh-pre-download/task.yaml

```diff
@@ -132,5 +136,6 @@ execute: |
     "$TESTSTOOLS"/snapd-state wait-for-snap-autorefresh "test-snapd-sh" "$OLD_CHANGE"
   else
     echo "unrecognized test variant"
+    tests.cleanup restore
```

Comment:
same as above

Reply:
This one wouldn't be a no-op because here the process is actually running (although if we run this case the test is really broken so it shouldn't happen)